v3/src/models/formula/formula-types.ts

```diff
@@ -35,7 +35,9 @@ export type IFormulaDependency = ILocalAttributeDependency | IGlobalValueDepende
 
 export type FValue = string | number | boolean
 
-export type EvaluateFunc = (...args: FValue[]) => FValue | FValue[]
+export type EvaluateFunc = (...args: FValue[]) => FValue
+
+export type EvaluateFuncWithAggregateContextSupport = (...args: (FValue | FValue[])[]) => FValue | FValue[]
```

Comment:
Defining a type like:
```typescript
export type FValueOrArray = FValue | FValue[]
```
would eliminate some redundancy here.

Reply:
Thanks, added in https://github.com/concord-consortium/codap/pull/1054/commits/afed1d74c66cb3fa37e4e7aa624eff5cc3194055